backend/src/main/java/corea/auth/interceptor/AuthorizationInterceptor.java

```diff
@@ -42,7 +44,16 @@ private boolean checkAnnotation(Object handler) {
 
         HandlerMethod handlerMethod = (HandlerMethod) handler;
 
-        return (null != handlerMethod.getMethodAnnotation(AUTH_ANNOTATION) ||
-                null != handlerMethod.getBeanType().getAnnotation(AUTH_ANNOTATION));
+        if (null != handlerMethod.getMethodAnnotation(AUTH_ANNOTATION) ||
+                null != handlerMethod.getBeanType().getAnnotation(AUTH_ANNOTATION)) {
+            return true;
+        }
+
+        for (Parameter parameter : handlerMethod.getMethod().getParameters()) {
+            if (parameter.isAnnotationPresent(AUTH_ANNOTATION)) {
+                return true;
+            }
+        }
+        return false;
```

Comment:
getParameters()가 어떤 타입으로 나오는지는 모르겠지만,
```suggestion
var parameters = handlerMethod.getMethod().getParameters();
return parameters.stream
             .anyMatch(parameter -> parameter.isAnnotationPresent(AUTH_ANNOTATION));
```
이러면 깔끔할거같네요~

Reply:
getParameters()의 반환 타입이 Parameter[] 라 이에 맞게 수정했습니다~